xivo_test_helpers/asset_launching_test_case.py

```diff
@@ -248,5 +265,7 @@ def _run_cmd(cmd, stderr=True):
     logger.debug('%s', cmd)
     stderr = subprocess.STDOUT if stderr else None
     completed_process = subprocess.run(cmd, stdout=subprocess.PIPE, stderr=stderr)
-    logger.info('%s', completed_process.stdout)
+    if completed_process.stdout:
+        for line in str(completed_process.stdout).replace('\\r', '').split('\\n'):
+            logger.info('stdout: %s', line)
```

Comment:
Have you a concrete example of the behavior that change?

Reply:
the output of the commands was on a single line but it was just for me. I'll revert that too.

Reply:
Before
```
xivo_test_helpers.asset_launching_test_case: DEBUG: ['docker-compose', '--no-ansi', '--project-name', 'confd', '--file', '/home/francois/dev/confd/integration_tests/suite/helpers/../../assets/docker-comp$
se.yml', '--file', '/home/francois/dev/confd/integration_tests/suite/helpers/../../assets/docker-compose.base.override.yml', 'down', '--timeout', '0', '--volumes']
xivo_test_helpers.asset_launching_test_case: INFO: b'Removing confd_sysconfd_1 ... \r\nRemoving confd_auth_1     ... \r\nRemoving confd_rabbitmq_1 ... \r\nRemoving confd_ari_1      ... \r\nRemoving confd$
confd_1    ... \r\nRemoving confd_provd_1    ... \r\nRemoving confd_postgres_1 ... \r\nRemoving confd_confd_1    ... done\r\nRemoving confd_rabbitmq_1 ... done\r\nRemoving confd_ari_1      ... done\r\nRe$
oving confd_sysconfd_1 ... done\r\nRemoving confd_provd_1    ... done\r\nRemoving confd_auth_1     ... done\r\nRemoving confd_postgres_1 ... done\r\nRemoving network confd_default\n'
```